app/packages/state/src/recoil/selectors.ts

```diff
@@ -40,6 +40,19 @@ export const mediaType = selector({
   },
 });
 
+export const savedViewsSelector = selector({
+  key: "datasetViews",
+  get: ({ get }) => {
+    console.log("dataset", get(atoms.dataset));
+    const savedViews = get(atoms.dataset)?.savedViews;
```

Comment:
Can we add `savedViews` to the dataset type in "../types.ts"? The selector should have implicit typing then.

Reply:
cool, if you mean to add a SavedView type in ./types and have the selector be typed for its return type - done!